repo_resource/common.py

```diff
@@ -94,23 +94,41 @@ def getRevision(remote, remoteUrl, project, branch):
     with git ls-remote command for each project
     without downloading the whole repo
     """
-    # v1.0^{} is the commit referring to tag v1.0
-    # git ls-remote returns the tag sha1 if left as is
-    if branch.startswith('refs/tags'):
-        branch += '^{}'
     try:
         with redirect_stdout(sys.stderr):
-            # return tuple (remote/project, revision)
-            print('Fetching revision for {}/{}...'.format(remote, project))
+            # return tuple (remote/project, branch, revision)
```

Comment:
I've turned this into an issue instead, to continue the discussion.

Since this has been approved, I'll go ahead and merge and draft a new minor release.

@massonju please continue discussion the review comments here or in the dedicated issue: https://github.com/makohoek/repo-resource/issues/42